misc/python/materialize/mzcompose.py

```diff
@@ -798,6 +798,23 @@ def __init__(
         )
 
 
+class SqlServer(PythonService):
+    def __init__(
+        self,
+        name: str = "sql-server",
+        image: str = "mcr.microsoft.com/mssql/server",
+        port: int = 1433,
+        sa_password: str = "j6pP@EoKFapawmx@Qy32",
```

Comment:
I am going ot get the password to be specified only once regardless of where it is used -- python or testdrive